recipe/meta.yaml

```diff
@@ -53,13 +52,14 @@ test:
         - celery.utils
         - celery.utils.dispatch
         - celery.worker
-        
+
     commands:
         - celery --version
         - celery worker --help
         - celery beat --help
         - celery multi help
         - celery shell --help
+        - pytest t/unit  # [py3k]
```

Comment:
I thought about leaving just the straight "import" test and the commands, but because of the conversation in conda-forge/staged-recipes#936 I thought we were better off running at least the unit-tests.

I will do what the majority agrees :)